### ISSUE TITLE
Allow CLI to retrieve more than 100 000 events

### ISSUE THREAD

**Issue by uhppoted** [uhppoted/uhppote-cli#7]
> Hi there,
>  
> I have been using uhppote-cli to connect to some controllers and I thought I would share with you my findings on the 100,000 event log buffer of these devices.
> I have a device that is currently at 111,545 events and still working, so I have been modifying the ROLLOVER variable in –cli in a fork in an attempt to work out the actual behaviour of the device, at the moment I can still access all events from 1 through 111545 from CLI or the web interface.
> I don’t know whether this is specific to a board version but this controller is running V7.70.61227
> I do have other boards with different versions of the firmware but they are not at 100,000 events yet.

**Comment by uhppoted** [comment]
From experimentation, it appears that the on-device event buffer is approximately 200 000 events:
```
Fri Dec 24 14:16:01 PST 2021
EVENTS:  423187757  1  186809
FIRST:   423187757  1      2019-02-10 07:12:01 6154410      3 false 6
LAST:    423187757  186809 2021-12-24 14:16:01 3922570474   4 true  44

Fri Dec 24 17:15:49 PST 2021
EVENTS:  423187757  2049  206817
FIRST:   423187757  2049   2021-12-17 11:42:55 3922570474   4 true  44
LAST:    423187757  206817 2021-12-24 17:15:48 3922570474   4 true  44
```

Once the on-device event buffer is full, events are dropped in chunks of 2048 events and the `first` event index updated accordingly i.e. the event index does not wrap around but continues to increment:
```
Fri Dec 24 17:15:49 PST 2021
EVENTS: 423187757  2049  206817
FIRST:  423187757  2049   2021-12-17 11:42:55 3922570474   4 true  44
LAST:   423187757  206817 2021-12-24 17:15:48 3922570474   4 true  44

Fri Dec 24 17:19:37 PST 2021
EVENTS: 423187757  2049   2021-12-17 11:42:55 3922570474   4 true  44
FIRST:  423187757  206825 2021-12-24 17:19:37 3922570474   4 true  44
LAST:   423187757  2049  206825
```

**Comment by uhppoted** [comment]
[events.log](https://github.com/uhppoted/uhppote-cli/files/7775347/events.log)

**Comment by uhppoted** [comment]
Reworked throughout - pending release of 0.7.2

**Comment by uhppoted** [comment]
Closing - fixed in release v0.7.2
